fuel-vm/src/interpreter/alu/wideint.rs

```diff
@@ -89,6 +89,8 @@ macro_rules! wideint_ops {
                     };
 
                     *dest = [<cmp_ $t:lower>](lhs, rhs, args.mode);
+                    *of = 0;
+                    *err = 0;
```

Comment:
If I remove these lines, tests still passes

Reply:
Can confirm, but I don't have any idea why. 
I have checked, and log does not reset the registers, so the only option is that the registers are reset already somewhere while executing the wideint compare instruction. Will check more in depth

Reply:
It's because code generated by `make_u128` and `make_u256` used in the tests reset `$err` and `$of` as they use `movi` internally.

You should first create all values, and only after that call the opcodes that access flags.

Reply:
I wish I looked at this comment three hours ago :)

Reply:
fixed tests in 18b60082a9285bb9a34ebc4ef962976513eb5a64 and verified that they fail before applying the changes to the wideint cmp alu operation